### PR TITLE
Update readme.txt

### PR DESCRIPTION
**Pull request by kevinbates**
Changed links from WooCommerce.com to Woo.com.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1.
2.
3.

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [x] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

This updates references in the plugin `readme.txt` file but does not change any functionality.

</details>

### CODE REVIEW

**Review by barryhughes — COMMENTED** [review]
Looks good! Random sampled some of the URLs and didn't find any problems. Can you confirm if we also need updates on the following lines?

- [Line 91](https://github.com/woocommerce/woocommerce/blob/9c646eb62b06be3378d50cf31f103e20ad3ee27b/plugins/woocommerce/readme.txt#L91)
- [Line 93](https://github.com/woocommerce/woocommerce/blob/9c646eb62b06be3378d50cf31f103e20ad3ee27b/plugins/woocommerce/readme.txt#L93)
- [Line 95](https://github.com/woocommerce/woocommerce/blob/9c646eb62b06be3378d50cf31f103e20ad3ee27b/plugins/woocommerce/readme.txt#L93)

**Comment by kevinbates** [comment]
Thanks @barryhughes !

Line 91 should read:

Where can I get help for extensions I have purchased from the WooCommerce Marketplace?

Line 93 needs to change to:

For assistance with paid extensions from the WooCommerce Marketplace: first, review our [self-service troubleshooting guide](https://woo.com/document/woocommerce-self-service-guide/). If the problem persists, kindly log a support ticket via [our helpdesk](https://woo.com/my-account/create-a-ticket/). Our dedicated Happiness Engineers aim to respond within 24 hours.

And 95:

I’m having trouble logging in to Woo.com – what now?

I'm not all that well versed in GitHub - mind making those changes for me? Thanks!

File: plugins/woocommerce/readme.txt [hunk 1]
```diff
@@ -80,45 +80,45 @@ WooCommerce is translated into multiple languages, including Danish, Ukrainian,
...
+If you get stuck, you can ask for help in the [WooCommerce Support Forum](https://wordpress.org/support/plugin/woocommerce/?utm_medium=referral&utm_source=wordpress.org&utm_campaign=wp_org_repo_listing) by following [these guidelines](https://wordpress.org/support/topic/guide-to-the-woocommerce-forum/?utm_medium=referral&utm_source=wordpress.org&utm_campaign=wp_org_repo_listing), reach out via the [WooCommerce Community Slack](https://woo.com/community-slack/?utm_medium=referral&utm_source=wordpress.org&utm_campaign=wp_org_repo_listing), or post in the [WooCommerce Community group](https://www.facebook.com/groups/advanced.woocommerce?utm_medium=referral&utm_source=wordpress.org&utm_campaign=wp_org_repo_listing) on Facebook.
 
 = Where can I get help for extensions I have purchased on WooCommerce.com? =
```
Comment (barryhughes) [inline plugins/woocommerce/readme.txt]: Updating per [this discussion](https://github.com/woocommerce/woocommerce/pull/41167#issuecomment-1789376834): 

```suggestion
= Where can I get help for extensions I have purchased from the WooCommerce Marketplace? =
```

File: plugins/woocommerce/readme.txt [hunk 2]
```diff
@@ -80,45 +80,45 @@ WooCommerce is translated into multiple languages, including Danish, Ukrainian,
...
 
-= Where can I get help for extensions I have purchased on WooCommerce.com? =
+= Where can I get help for extensions I have purchased from the WooCommerce Marketplace? =
 
-For assistance with paid extensions from the WooCommerce.com Marketplace: first, review our [self-service troubleshooting guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/). If the problem persists, kindly log a support ticket via [our helpdesk](https://woocommerce.com/my-account/create-a-ticket/). Our dedicated Happiness Engineers aim to respond within 24 hours.
+For assistance with paid extensions from the WooCommerce.com Marketplace: first, review our [self-service troubleshooting guide](https://woo.com/document/woocommerce-self-service-guide/). If the problem persists, kindly log a support ticket via [our helpdesk](https://woo.com/my-account/create-a-ticket/). Our dedicated Happiness Engineers aim to respond within 24 hours.
```
Comment (barryhughes) [inline plugins/woocommerce/readme.txt]: Updating per [this discussion](https://github.com/woocommerce/woocommerce/pull/41167#issuecomment-1789376834):

```suggestion
For assistance with paid extensions from the WooCommerce Marketplace: first, review our [self-service troubleshooting guide](https://woo.com/document/woocommerce-self-service-guide/). If the problem persists, kindly log a support ticket via [our helpdesk](https://woo.com/my-account/create-a-ticket/). Our dedicated Happiness Engineers aim to respond within 24 hours.
```

File: plugins/woocommerce/readme.txt [hunk 3]
```diff
@@ -80,45 +80,45 @@ WooCommerce is translated into multiple languages, including Danish, Ukrainian,
...
 
-= Where can I get help for extensions I have purchased on WooCommerce.com? =
+= Where can I get help for extensions I have purchased from the WooCommerce Marketplace? =
 
-For assistance with paid extensions from the WooCommerce.com Marketplace: first, review our [self-service troubleshooting guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/). If the problem persists, kindly log a support ticket via [our helpdesk](https://woocommerce.com/my-account/create-a-ticket/). Our dedicated Happiness Engineers aim to respond within 24 hours.
+For assistance with paid extensions from the WooCommerce Marketplace: first, review our [self-service troubleshooting guide](https://woo.com/document/woocommerce-self-service-guide/). If the problem persists, kindly log a support ticket via [our helpdesk](https://woo.com/my-account/create-a-ticket/). Our dedicated Happiness Engineers aim to respond within 24 hours.
 
 = I’m having trouble logging in to WooCommerce.com – what now? =
```
Comment (barryhughes) [inline plugins/woocommerce/readme.txt]: Updating per [this discussion](https://github.com/woocommerce/woocommerce/pull/41167#issuecomment-1789376834):

```suggestion
= I’m having trouble logging in to Woo.com – what now? =
```

**Review by barryhughes — APPROVED** [review]

**Comment by barryhughes** [comment]
Thanks, we can merge once checks pass 👍🏼

**Comment by lsinger** [comment]
Closing and re-opening to see whether that makes the tests start. 🙄

**Review by lsinger — APPROVED** [review]
Looks good to me as well. 🚢

**Pull request merged**